stud.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |spec|
     files += Dir["#{dir}/**/*"]
   end
 
-  files << "LICENSE" 
+  files << "LICENSE"
```

Comment:
Rest in peace, oh trailing white space. Nobody noticed you were there. Nobody will notice when you are gone. Nobody will notice when I resurrect you later by accident.